### PR TITLE
Corrected ojdkbuild.ojdkbuild 1.8.3321.9 version number.

### PR DESCRIPTION
**Pull request by emilkloeden**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----
In my previous PR I incorrectly had an extra '0' in my versioning, meaning the latest version was not being reported as the latest version. This PR addresses that by moving 1.8.0.3321.9 to 1.8.3321.9 and updating the designation in each .yaml file.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/91717)

### CODE REVIEW

File: manifests/o/ojdkbuild/ojdkbuild/1.8.3321.9/ojdkbuild.ojdkbuild.installer.yaml [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
+
+PackageIdentifier: ojdkbuild.ojdkbuild
+PackageVersion: 1.8.3321.9
+MinimumOSVersion: 10.0.0.0
+Installers:
+- Architecture: x64
+  InstallerType: wix
+  InstallerUrl: https://github.com/ojdkbuild/ojdkbuild/releases/download/java-1.8.0-openjdk-1.8.0.332-1.b09/java-1.8.0-openjdk-1.8.0.332-1.b09.ojdkbuild.windows.x86_64.msi
+  InstallerSha256: 84436C850A12F52C9658449D5BC16F1F6AD1C0AF820D510414A45B8D8AD64357
+  ProductCode: '{F115813D-E05C-48A3-9F42-5C637056203B}'
```
Comment (mdanish-kh) [inline manifests/o/ojdkbuild/ojdkbuild/1.8.3321.9/ojdkbuild.ojdkbuild.installer.yaml:12]:
```suggestion
  ProductCode: '{F115813D-E05C-48A3-9F42-5C637056203B}'
  AppsAndFeaturesEntries:
  - DisplayName: OpenJDK 1.8.0_332-1-ojdkbuild
    ProductCode: '{F115813D-E05C-48A3-9F42-5C637056203B}'
```
For better upgrade matching

File: manifests/o/ojdkbuild/ojdkbuild/1.8.3321.9/ojdkbuild.ojdkbuild.installer.yaml [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
+- Architecture: x86
+  InstallerType: wix
+  InstallerUrl: https://github.com/ojdkbuild/ojdkbuild/releases/download/java-1.8.0-openjdk-1.8.0.332-1.b09-x86/java-1.8.0-openjdk-1.8.0.332-1.b09.ojdkbuild.windows.x86.msi
+  InstallerSha256: 048b5de7c319a5989465b6d734f5f7a5a7a1eb0881f89e19473abe59fbd9c574
+  ProductCode: '{30DF20B4-B251-4DCF-AE84-59296ED8CB8D}'
```
Comment (mdanish-kh) [inline manifests/o/ojdkbuild/ojdkbuild/1.8.3321.9/ojdkbuild.ojdkbuild.installer.yaml:17]:
```suggestion
  ProductCode: '{30DF20B4-B251-4DCF-AE84-59296ED8CB8D}'
  AppsAndFeaturesEntries:
  - DisplayName: OpenJDK 1.8.0_332-1-ojdkbuild-x86
    ProductCode: '{30DF20B4-B251-4DCF-AE84-59296ED8CB8D}'
```
For better upgrade matching

File: manifests/o/ojdkbuild/ojdkbuild/1.8.3321.9/ojdkbuild.ojdkbuild.locale.en-US.yaml [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
-
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: ojdkbuild.ojdkbuild
+PackageVersion: 1.8.3321.9
+PackageLocale: en-US
+Publisher: ojdkbuild open-source project
+PackageName: ojdkbuild OpenJDK JRE 8
+PackageUrl: https://github.com/ojdkbuild/ojdkbuild/wiki/Motivation
```
Comment (mdanish-kh) [inline manifests/o/ojdkbuild/ojdkbuild/1.8.3321.9/ojdkbuild.ojdkbuild.locale.en-US.yaml:9]:
```suggestion
PackageUrl: https://github.com/ojdkbuild/ojdkbuild/
```
Wiki can be mentioned in the `Documentations` section.

File: manifests/o/ojdkbuild/ojdkbuild/1.8.3321.9/ojdkbuild.ojdkbuild.locale.en-US.yaml [hunk 4]
```diff
@@ -1,15 +1,15 @@
...
+License: GPL 2 with Classpath Exception
+LicenseUrl: https://github.com/ojdkbuild/ojdkbuild/blob/master/LICENSE
+ShortDescription: Community builds using source code from OpenJDK project
```
Comment (mdanish-kh) [inline manifests/o/ojdkbuild/ojdkbuild/1.8.3321.9/ojdkbuild.ojdkbuild.locale.en-US.yaml:12]:
```suggestion
ShortDescription: Community builds using source code from OpenJDK project
Documentations:
- DocumentLabel: Wiki
  DocumentUrl: https://github.com/ojdkbuild/ojdkbuild/wiki/
```

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

**Comment by emilkloeden** [comment]
Thanks @mdanish-kh for the feedback. I've accepted, batched all the suggested changes and I think committed it. Either way it's triggered some CI/CD things. Hopefully that's enough to get the registry updated. Let me know if there's anything else I should do. Otherwise, thank you again :).

**Review by mdanish-kh — APPROVED** [review]
LGTM, thank you.

**Pull request merged**